modules/installation-user-infra-machines-iso.adoc

```diff
@@ -43,61 +47,47 @@ You must download images with the highest version that is less than or equal
 to the {product-title} version that you install. Use the image versions
 that match your {product-title} version if they are available.
 Only use ISO images for this procedure.
-RHCOS qcow2 images are not supported for bare metal installs.
+{op-system} qcow2 images are not supported for bare metal installs.
 ====
 +
-You must download the ISO file and the RAW disk file.
-Those file names resemble the following examples:
+ISO file names resemble the following example:
 
-** ISO: `rhcos-<version>-installer.<architecture>.iso`
-** Compressed metal RAW: `rhcos-<version>-metal.<architecture>.raw.gz`
+`rhcos-<version>-installer.<architecture>.iso`
```

Comment:
The indenting in the preview build is off. Might need a `+` above this file name example.